### ISSUE TITLE
NTR: Cardiac imaging: cardiac mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8901]
New term request Cardiac imaging: cardiac mass (elthat_100319043302): Echocardiogram or CT; cardiac ultrasound; heart  UMLS: 3670832

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000847
- label: Cardiac mass
- parents: [Abnormal heart morphology (HP:0001627)](https://hpo.jax.org/app/browse/term/HP:0001627)
- pmid: 
- def: An abnormal enlargement or swelling localized in the heart. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8901
- synonyms:
